entity/boot_resource_test.go

```diff
@@ -0,0 +1,80 @@
+package entity
+
+import (
+	"testing"
+
+	"github.com/canonical/gomaasclient/test/helper"
+	"github.com/google/go-cmp/cmp"
+)
+
+var sampleBootResource BootResource = BootResource{
+	Sets: map[string]BootResourceSet{
+		"20240828": {
```

Comment:
where does 20240828 come from?

Reply:
I think this is coming from the upload date of the custom images. I took it from my dev maas